datalogistik/util.py

```diff
@@ -284,16 +287,108 @@ def schema_to_dict(schema):
     return field_dict
 
 
+def convert_arrow_alias(type_name):
+    aliases = {
+        "bool": "bool_",
+        "halffloat": "float16",
+        "float": "float32",
+        "double": "float64",
+        "decimal": "decimal128",
+    }
+    for (alias, func_name) in aliases.items():
+        if type_name == alias:
+            return func_name
+    # no alias was found
+    return type_name
```

Comment:
nit: you could reduce some complexity here!
```suggestion
    return aliases.get(type_name, default=type_name)
```

Reply:
Thanks! much better